sample/lib/src/commonMain/kotlin/software/amazon/lastmile/kotlin/inject/anvil/sample/AssistedIdProvider.kt

```diff
@@ -0,0 +1,26 @@
+package software.amazon.lastmile.kotlin.inject.anvil.sample
+
+import me.tatarka.inject.annotations.Assisted
+import me.tatarka.inject.annotations.Inject
+import software.amazon.lastmile.kotlin.inject.anvil.AppScope
+import software.amazon.lastmile.kotlin.inject.anvil.ContributesBinding
+import software.amazon.lastmile.kotlin.inject.anvil.SingleIn
+
+/**
+ * API to get the application ID. This is used in assisted injection scenarios to provide the
+ * application ID to the object graph with an extra parameter provided at runtime.
+ */
+interface AssistedApplicationIdProvider {
```

Comment:
Could you please remove this file again? I'd not like to update the sample, instead you should cover this use case through tests in the `:compiler` module.